messaging/src/main/java/org/jboss/as/messaging/TransportConfigOperationHandlers.java

```diff
@@ -145,6 +145,16 @@ static void processConnectors(final OperationContext context, final Configuratio
                 final ModelNode config = property.getValue();
                 final Map<String, Object> parameters = getParameters(context, config);
                 final String clazz = FACTORY_CLASS.resolveModelAttribute(context, config).asString();
+                if (NettyConnectorFactory.class.getName().equals(clazz)) {
```

Comment:
Hi Jeff, as far as I understand, if user use connector with netty factory class, it is the same as the netty-connector. However connectors can have factory class other then netty. Here we need to check if the factory is netty or not.